setup.py

```diff
@@ -5,6 +5,7 @@
 setup(name='python-meh', version='0.48',
       description='Python module for handling exceptions',
       author='Chris Lumens', author_email='clumens@redhat.com',
-      url='http://git.fedoraproject.org/git/?p=python-meh.git',
-      data_files = [('/usr/share/python-meh', ['ui/exception-dialog.glade'])],
-      packages=['meh', 'meh.ui'])
+      url='https://github.com/rhinstaller/python-meh',
+      data_files=[('/usr/share/python-meh', ['ui/exception-dialog.glade'])],
+      packages=['meh', 'meh.ui'],
+      install_requires=['report', 'pydbus'])
```

Comment:
I think there should me more requirements than just these two based on spec file.

Reply:
Changed - now there's everything.